infrastructure/oidc.tf

```diff
@@ -16,11 +16,11 @@ module "gh_oidc" {
     data.aws_iam_policy.adm.arn
   ]
   # add extra conditions, will be merged with the default_conditions
-  # conditions = [{
-  #   test     = "StringLike"
-  #   variable = "token.actions.githubusercontent.com:sub"
-  #   values   = [
-  #     "repo:master-the-legacy/master-the-legacy:pull_request",
-  #   ]
-  # }]
+  conditions = [{
+    test     = "StringLike"
+    variable = "token.actions.githubusercontent.com:sub"
+    values = [
+      "repo:master-the-legacy/votingapp-results:*",
```

Comment:
shouldn't we add the gitops repository here as well?

Reply:
It is here https://github.com/master-the-legacy/gitops/pull/15/files#diff-fb257527df8288b3889a076b4fdd34fcf36d241aa0939ebe30d5fd5f6286eb33R9 by default